datasets/0.8/fashion-mnist/metadata.json

```diff
@@ -96,4 +96,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Can you keep the end-of-line?

Reply:
sure sure. that's a bit a tricky one. VSC removes this automatically when saving the file unless I set the following 
![Screenshot 2024-02-12 at 20 05 54](https://github.com/mlcommons/croissant/assets/87860698/7da49f40-2960-4d21-8068-155cf8baba8e)

Reply:
Yes! Can you please set this option?